client/stylesheets/_toCat.scss

```diff
@@ -2060,6 +2065,16 @@ i:hover {
     margin: 3rem 0;
 }
 
+.helpImgUl li {
+    width: 25%;
+}
+
+.aboutImg {
+    width: 100% !important;
```

Comment:
!important should not be used<br>Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered height, max-height, width